### PR TITLE
Add Razer DeathAdder Essential

### PR DESCRIPTION
**Pull request by lsdage**
Initial support for Razer DeathAdder Essential - USB VID:PID 1532:006E
See also: #699

What works:
    Setting DPI
    Setting logo brightness
    Setting logo effect (none|breathing|static)
    Setting scroll wheel brightness
    Setting scroll wheel (none|breathing|static)

Needs testing:
    Setting Poll rate

Known issues:

RazorGenie offers the color selection for logo scroll wheel. Setting this affects the color but the mouse only changes the amount of green. This feature would presumably if the mouse had all LEDs (not just a green one).

Setting DPI for Y axle has no effect. Not sure if the mouse supports seperate DPIs for X and Y axles. Probably not.

### CODE REVIEW

**Comment by z3ntu** [comment]
Thanks for the PR!

> Raz~~o~~erGenie offers the color selection for logo scroll wheel.

Support for non-RGB LEDs isn't really present neither in openrazer nor RazerGenie, as 99% of Razer devices have RGB lighting...

> Setting DPI for Y axle has no effect.

That sounds weird; but as this is the "budget mouse" from Razer I can imagine they used a cheap sensor that doesn't have a separate x/y resolution

File: daemon/openrazer_daemon/hardware/mouse.py [hunk 1]
```diff
@@ -1073,3 +1073,59 @@ def _resume_device(self):
...
+
+    # Deprecated - RAZER_URLS be removed in future.
+    RAZER_URLS = {
+        "top_img": "https://assets.razerzone.com/eeimages/support/products/1385/1385_deathadderessential.png",
+        "side_img": None,
+        "perspective_img": None
+    }
+
+    #def __init__(self, *args, **kwargs):
+    #    super(RazerDeathAdderEssential, self).__init__(*args, **kwargs)
```
Comment (z3ntu) [inline daemon/openrazer_daemon/hardware/mouse.py]: Remove the commented code if it isn't used

File: daemon/openrazer_daemon/hardware/mouse.py [hunk 2]
```diff
@@ -1073,3 +1073,59 @@ def _resume_device(self):
...
+
+    DPI_MAX = 6400
+
+    DEVICE_IMAGE = "https://assets.razerzone.com/eeimages/support/products/1385/1385_deathadderessential.png"
+
+    # Deprecated - RAZER_URLS be removed in future.
+    RAZER_URLS = {
+        "top_img": "https://assets.razerzone.com/eeimages/support/products/1385/1385_deathadderessential.png",
+        "side_img": None,
+        "perspective_img": None
```
Comment (z3ntu) [inline daemon/openrazer_daemon/hardware/mouse.py]: Remove the lines with `None`

File: README.md [hunk 3]
```diff
@@ -1,8 +1,34 @@
...
+- Setting DPI for Y axle has no effect. Not sure if the mouse supports seperate DPIs for X and Y axles.
+
 # [OpenRazer](https://openrazer.github.io/)
 
 A collection of Linux drivers for the Razer devices, providing kernel drivers, DBus services and python bindings to interact with the DBus interface.
 
-**Before raising an issue** saying something doesn't work, read [the troubleshooting wiki page](https://github.com/openrazer/openrazer/wiki/Troubleshooting), try not to create new issues if one exists.
+**Before raising an issue** saying sogit push --set-upstream origin DeathAdder_Essentialmething doesn't work, read [the troubleshooting wiki page](https://github.com/openrazer/openrazer/wiki/Troubleshooting), try not to create new issues if one exists.
 
 Also this is the master branch, devices may appear in the tables below but might not be released in package form yet. Check out the stable branch for what _should_ be in the packages.
```
Comment (z3ntu) [inline README.md]: For the PR, please revert the readme changes and add the device line into the correct section

**Review by z3ntu — CHANGES_REQUESTED** [review]
LGTM, just a few comments

**Comment by lsdage** [comment]
Thank you for the review and tips.
Code has been cleaned up.

**Review by z3ntu — APPROVED** [review]
LGTM

**Pull request merged**